roachtest: django failed

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4913468&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4913468&tab=artifacts#/django) on master @ [dcb06e12a77c0a7f7fe9d2609c54327345987dcd](https://github.com/cockroachdb/cockroach/commits/dcb06e12a77c0a7f7fe9d2609c54327345987dcd):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/django/run_1
	orm_helpers.go:190,orm_helpers.go:116,django.go:227,django.go:238,test_runner.go:875: 
		Tests run on Cockroach v22.1.0-alpha.4-987-gdcb06e12a7
		Tests run against django cockroach-3.2.x
		9043 Total Tests Run
		9042 tests passed
		1 test failed
		612 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: queries.test_explain.ExplainTests.test_unknown_options (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist22_1) is available in the artifacts' django log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #80048 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #80042 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.1 release-blocker]
- #79989 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1 release-blocker]
- #79988 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1.0 release-blocker]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4916356&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4916356&tab=artifacts#/django) on master @ [6c91da48b8da78c13e4cb8675a1a98c5ec75ea06](https://github.com/cockroachdb/cockroach/commits/6c91da48b8da78c13e4cb8675a1a98c5ec75ea06):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/django/run_1
	orm_helpers.go:190,orm_helpers.go:116,django.go:227,django.go:238,test_runner.go:875: 
		Tests run on Cockroach v22.1.0-alpha.4-989-g6c91da48b8
		Tests run against django cockroach-3.2.x
		9043 Total Tests Run
		9042 tests passed
		1 test failed
		612 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: queries.test_explain.ExplainTests.test_unknown_options (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist22_1) is available in the artifacts' django log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #80048 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #80042 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.1]
- #79989 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #79988 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4924120&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4924120&tab=artifacts#/django) on master @ [1fdbb16fa206c5fd77c3aa111ec40916edfb55df](https://github.com/cockroachdb/cockroach/commits/1fdbb16fa206c5fd77c3aa111ec40916edfb55df):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/django/run_1
	orm_helpers.go:190,orm_helpers.go:116,django.go:227,django.go:238,test_runner.go:875: 
		Tests run on Cockroach v22.1.0-alpha.4-1025-g1fdbb16fa2
		Tests run against django cockroach-3.2.x
		9043 Total Tests Run
		9042 tests passed
		1 test failed
		612 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: queries.test_explain.ExplainTests.test_unknown_options (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist22_1) is available in the artifacts' django log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #80048 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #80042 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.1]
- #79989 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #79988 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4935109&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4935109&tab=artifacts#/django) on master @ [7ea67fd5d4966508877fc88ba21bdd498ff162e8](https://github.com/cockroachdb/cockroach/commits/7ea67fd5d4966508877fc88ba21bdd498ff162e8):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/django/run_1
	orm_helpers.go:190,orm_helpers.go:116,django.go:227,django.go:238,test_runner.go:876: 
		Tests run on Cockroach v22.1.0-alpha.4-1061-g7ea67fd5d4
		Tests run against django cockroach-3.2.x
		9043 Total Tests Run
		9042 tests passed
		1 test failed
		612 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: queries.test_explain.ExplainTests.test_unknown_options (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist22_1) is available in the artifacts' django log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #80048 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #80042 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.1]
- #79989 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #79988 roachtest: django failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1.0]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/django.go
+++ b/pkg/cmd/roachtest/tests/django.go
@@ -27,8 +27,8 @@ import (
 var djangoReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)(\.(?P<point>\d+))?$`)
 var djangoCockroachDBReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)$`)
 
-var djangoSupportedTag = "cockroach-3.2.x"
-var djangoCockroachDBSupportedTag = "3.2.1"
+var djangoSupportedTag = "cockroach-4.0.x"
+var djangoCockroachDBSupportedTag = "4.0.1"
 
 func registerDjango(r registry.Registry) {
 	runDjango := func(

